src/dom-updates.js

```diff
@@ -1,44 +1,57 @@
 import {
   getBookings,
-  getTotalCost
+  getTotalCost,
+  searchByDate
 } from './bookings'
 
 import {
   yourBookings,
-  yourBookingsCost
+  yourBookingsCost,
+  dateField,
+  dashboard,
+  searchResults,
+  availableRooms
 } from './scripts'
 
 const setDashboard = (customer, allBookings, allRooms) => {
+  flatpickr(dateField, {
+    minDate: 'today',
+    altInput: true,
+    altFormat: "F j, Y",
+    dateFormat: "Y/m/d"
+  })
+  
   const usersBookings = getBookings(allBookings, customer);
-  const spent = getTotalCost(allRooms, usersBookings);
-
-  yourBookingsCost.innerHTML = '';
-  yourBookings.innerHTML = '';
-
-  yourBookingsCost.innerHTML = `$${spent}`
+  
+  if (typeof usersBookings === 'string') {
+    yourBookingsCost.innerHTML = `<p>${usersBookings}</p>`
+  } else {
+    const spent = getTotalCost(allRooms, usersBookings);
 
-  usersBookings.forEach(booking => {
-    let room = allRooms.find(room => room.number === booking.roomNumber);
-    let image = setRoomImage(room);
-    let bidet = checkForBidet(room);
+    yourBookingsCost.innerHTML = '';
```

Comment:
Since yourBookingsCost gets simply reassigned and not added to, you may be able to forgo this line altogether

Reply:
Great suggestion. Done.